config/locales/en.yml

```diff
@@ -884,11 +895,12 @@ headings.account.devices: Devices
 headings.account.events: Events
 headings.account.federal_employee_id: Federal employee ID
 headings.account.login_info: Your account
+headings.account.pending: Pending
 headings.account.reactivate: Reactivate your account
 headings.account.two_factor: Your authentication methods
 headings.account.unphishable: Unphishable
-headings.account.verified_account: Verified Account
```

Comment:
I know this existed already, but these don't really feel like "headings", since they're being used as badge content. This would be a good opportunity to rename it to something more appropriate.

Reply:
Rearranged keys in 9d16b861e4.